packages/core/src/client.ts

```diff
@@ -29,8 +29,6 @@ import {
   UserConfiguration,
   ValueSet,
 } from '@medplum/fhirtypes';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-/** @ts-ignore */
```

Comment:
This is necessary for `pdfmake` to be an optional dependency in TypeScript projects.

Reply:
Ah yeah I think auto-organize messed things up and put it somewhere it didn't make sense. I'll fix that

Reply:
Done